### PR TITLE
Implements fixed-user-size param and behavior.

### LINKED ISSUE #93: minimum user size
I would like to visualize a project which starts out small and eventually becomes quite big. So big that the user avatars are only a few pixels in size. There is ` --user-scale` to work with indeed, but setting this to a value that gives the avatars a good scaling towards the end (about `10` in my case) makes them ridiculously large when the animation starts.

A good way out would perhaps be a ` --min-avatar-size` parameter.

### PR DESCRIPTION
**Pull request by 0jrp0**
Resolves #93.

### CODE REVIEW

File: src/gource.h [hunk 1]
```diff
@@ -176,6 +176,7 @@ class Gource : public SDLApp {
     Bounds2D active_user_bounds;
 
     int commitqueue_max_size;
+    int starting_z;
```
Comment (acaudwell) [inline src/gource.h:179]: This should be a float.

File: src/gource.cpp [hunk 2]
```diff
@@ -901,6 +901,7 @@ void Gource::reset() {
     mousedragged = false;
 
     commitqueue_max_size = 100;
+    starting_z = -300;
```
Comment (acaudwell) [inline src/gource.cpp:904]: You need to move this into the constructor before ZoomCamera() gets called. I think its getting a random value right now.

File: src/gource.cpp [hunk 3]
```diff
@@ -2237,7 +2238,11 @@ void Gource::updateVBOs(float dt) {
             float alpha = user->getAlpha();
             vec3 col   = user->getColour();
 
-            user_vbo.add(user->graphic->textureid, user->getPos() - user->dims*0.5f, user->dims, vec4(col.x, col.y, col.z, alpha));
+            vec2 scaled_dims = user->dims;
+
```
Comment (acaudwell) [inline src/gource.cpp:2242]: there's an extra new line here

**Comment by acaudwell** [comment]
Other than the few things I noticed above, looks good. Maybe you could add the help text, as well as add it to the README and data/gource.1 (the man page). Thanks

**Comment by 0jrp0** [comment]
Pushed two commits that address comments. This is my first contrib to any github project ever. Let me know if I screwed up the workflow, etc.

**Pull request merged**